src/services/infra/InfraService.ts

```diff
@@ -91,4 +104,123 @@ export default class InfraService {
       throw err
     }
   }
+
+  launchSite = async (
+    submissionId: string,
+    requestor: User,
+    agency: User,
+    repoName: string,
+    primaryDomain: string,
+    subDomainSettings: SubDomainSettings
+  ) => {
+    // call amplify to trigger site launch process
+    try {
+      // Set up domain association using LaunchesService
+      const {
+        appId,
+        siteId,
+      } = await this.launchesService.configureDomainInAmplify(
+        repoName,
+        primaryDomain,
+        subDomainSettings
+      )
+      logger.info(
+        `Created Domain association for ${repoName} to ${primaryDomain}`
+      )
+
+      // Get DNS records from Amplify
+      const dnsInfo = await this.launchesService.getDomainAssociationRecord(
+        primaryDomain,
+        appId
+      )
+
+      const certificationRecord = dnsInfo.domainAssociation?.certificateVerificationDNSRecord?.split(
```

Comment:
Is it possible to add a comment to give an example of how the `certificationRecord` should look like?

Reply:
[be86a41](https://github.com/isomerpages/isomercms-backend/pull/517/commits/be86a41d7454613f57462ce63876b32150f86b0e)